File: EIDSS v5/android.java/workspace/EIDSS/src/com/bv/eidss/model/Options.java
package com.bv.eidss.model;

import android.database.Cursor;
import android.content.ContentValues;

public class Options {

	private long id; 
    private String strServerUrl;
    private String strLocalPassword;
    private String strLoginOrganization;
    private String strLoginUsername;

	public long getId() { return id; } public void setId(long val) { id = val; } 
	public String getServerUrl() { return strServerUrl; } public void setServerUrl(String val) { strServerUrl = val; } 
	public String getLocalPassword() { return strLocalPassword; } public void setLocalPassword(String val) { strLocalPassword = val; } 
	public String getLoginOrganization() { return strLoginOrganization; } public void setLoginOrganization(String val) { strLoginOrganization = val; } 
	public String getLoginUsername() { return strLoginUsername; } public void setLoginUsername(String val) { strLoginUsername = val; } 
    

    public static Options FromCursor(Cursor cursor)
    {
        Options ret = new Options();
        ret.id = cursor.getLong(cursor.getColumnIndex("id"));
        ret.strServerUrl = cursor.getString(cursor.getColumnIndex("strServerUrl"));
        ret.strLocalPassword = cursor.getString(cursor.getColumnIndex("strLocalPassword"));
        ret.strLoginOrganization = cursor.getString(cursor.getColumnIndex("strLoginOrganization"));
        ret.strLoginUsername = cursor.getString(cursor.getColumnIndex("strLoginUsername"));
        return ret;
    }

    public ContentValues ContentValues()
    {
    	ContentValues ret = new ContentValues();
        ret.put("id", id);
        ret.put("strServerUrl", strServerUrl);
        ret.put("strLocalPassword", strLocalPassword);
        ret.put("strLoginOrganization", strLoginOrganization);
        ret.put("strLoginUsername", strLoginUsername);
        return ret;
    }
	
}

File: EIDSS v5/android/eidssdroid/Model/Options.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

using Android.App;
using Android.Content;
using Android.Database;
using Android.OS;
using Android.Runtime;
using Android.Views;
using Android.Widget;

namespace eidssdroid.Model
{
    public class Options
    {
        public Int64 id { get; set; }
        public String strServerUrl { get; set; }
        public String strLocalPassword { get; set; }
        public String strLoginOrganization { get; set; }
        public String strLoginUsername { get; set; }


        #region FromCursor
        public static Options FromCursor(ICursor cursor)
        {
            return new Options()
            {
                id = cursor.GetLong(cursor.GetColumnIndex("id")),
                strServerUrl = cursor.GetString(cursor.GetColumnIndex("strServerUrl")),
                strLocalPassword = cursor.GetString(cursor.GetColumnIndex("strLocalPassword")),
                strLoginOrganization = cursor.GetString(cursor.GetColumnIndex("strLoginOrganization")),
                strLoginUsername = cursor.GetString(cursor.GetColumnIndex("strLoginUsername"))
            };
        }
        #endregion

        #region ContentValues
        public ContentValues ContentValues()
        {
            var ret = new ContentValues();
            ret.Put("id", id);
            ret.Put("strServerUrl", strServerUrl);
            ret.Put("strLocalPassword", strLocalPassword);
            ret.Put("strLoginOrganization", strLoginOrganization);
            ret.Put("strLoginUsername", strLoginUsername);
            return ret;
        }
        #endregion

    }
}